packages/xstate-wallet/notes/outcomes.md

```diff
@@ -0,0 +1,62 @@
+# Outcomes
+
+The wallet currently uses two types of outcomes.
+
+```typescript
+interface SimpleGuarantee {
+  type: 'SimpleGuarantee';
+  targetChannelId: string;
+  assetHolderAddress: string;
+  destinations: string[];
+}
+
+interface SimpleAllocation {
+  type: 'SimpleAllocation';
+  assetHolderAddress: string;
+  allocationItems: {destination: Destination; amount: BigNumber}[];
+}
+```
+
+The outcome determines what happens to funds in a channel, when the channel is finalized.
+This is explained in detail in the [nitro paper](https://magmo.com/nitro-protocol.pdf).
+
+- For simple allocations, `amount` is _transferred_ to `destination`
+  - In case the channel is _underfunded_ -- the holdings are less than the sum of `amount` over the allocation items -- allocation items are prioritized by their order
+- For simple guarantees, funds are transferred according to the outcome of the `targetChannelId`
+  - :warning: If the target channel does not have a simple allocation outcome, things will break.
+  - In case the guarantor channell is _underfunded_ -- the holdings are less than the sum of `amount` over the allocation items of `guarantee.targetChannelId` -- allocation items are prioritized by the order in `guarantee.destinations`
```

Comment:
```suggestion
  - In case the guarantor channel is _underfunded_ -- the holdings are less than the sum of `amount` over the allocation items of `guarantee.targetChannelId` -- allocation items are prioritized by the order in `guarantee.destinations`
```